juju-charms/charm-trilio-data-mover/src/reactive/trilio_data_mover.py

```diff
@@ -208,34 +202,35 @@ def create_virt_env():
     try:
         # remove old venv if it exists
         os.system('rm -rf {}'.format(venv_path))
-        venv_src = 'http://{}:8081/packages/queens_ubuntu'\
-                   '/tvault-contego-virtenv.tar.gz'.format(tv_ip)
-        venv_dest = path
-        handler.install(venv_src, venv_dest)
+        venv_dest = '/'
+        handler.install(venv_url, venv_dest)
         log("Virtual Environment installed successfully")
     except Exception as e:
         log("Failed to install Virtual Environment")
         return False
 
+    # Install TrilioVault Datamover package
+    if not install_plugin(pkg_name):
+        return False
+
     # Get dependent libraries paths
     try:
-        cmd = ['/usr/bin/python', 'files/trilio/get_pkgs.py']
+        cmd = ['/usr/bin/python{}'.format(config('python-version')),
```

Comment:
Will this work if 'python-version' = 2? it will be /usr/bin/python2

Reply:
yes, python2 will be present if the interpreter is available. It is symlink to python2.7, same as python binary.